src/Apps/Order2/Routes/Checkout/Components/PaymentStep/__tests__/Order2PaymentForm.jest.tsx

```diff
@@ -383,6 +521,16 @@ describe("Order2PaymentForm", () => {
         mockCheckoutContext.checkoutTracking.clickedOrderProgression,
       ).toHaveBeenCalledWith("ordersPayment")
 
+      // Expect the confirmation token query to be called
+      await waitFor(() => {
+        expect(mockFetchQuery).toHaveBeenCalledWith(
+          expect.anything(), // environment
+          expect.anything(), // graphql query
```

Comment:
then here you can use the result of `mockResolveLastOperation` to assert on the operation, its name and variables. I think this should work for both queries and mutations but you might need to double check the implementation of that helper function if not - it just wraps the mock relay environment's resolver.